v2/pkg/apis/kubeflow/v2beta1/swagger.json

```diff
@@ -247,6 +167,87 @@
           "type": "string"
         }
       }
+    },
+    "v2beta1.MPIJob": {
+      "type": "object",
+      "properties": {
+        "apiVersion": {
+          "description": "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources",
+          "type": "string"
+        },
+        "kind": {
+          "description": "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds",
+          "type": "string"
+        },
+        "metadata": {
+          "$ref": "#/definitions/v1.ObjectMeta"
+        },
+        "spec": {
+          "$ref": "#/definitions/v2beta1.MPIJobSpec"
+        },
+        "status": {
+          "$ref": "#/definitions/v1.JobStatus"
+        }
+      }
+    },
+    "v2beta1.MPIJobList": {
+      "type": "object",
+      "required": [
+        "metadata",
+        "items"
+      ],
+      "properties": {
+        "apiVersion": {
+          "description": "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources",
+          "type": "string"
+        },
+        "items": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/v2beta1.MPIJob"
+          }
+        },
+        "kind": {
+          "description": "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds",
+          "type": "string"
+        },
+        "metadata": {
+          "$ref": "#/definitions/v1.ListMeta"
+        }
+      }
+    },
+    "v2beta1.MPIJobSpec": {
+      "type": "object",
+      "required": [
+        "runPolicy",
```

Comment:
This shouldn't be required... are we missing an annotation? (like +optional omitempty or something like that)

Reply:
Yes compared to v1 need add the `omitempty` annotation

Reply:
Please add it. Since it's not required in the CRD definition, it should be a backwards compatible change https://github.com/kubeflow/mpi-operator/blob/285cb98d5985ff8e8a03384929a58cf502b26375/manifests/base/crd.yaml#L103

Reply:
OK, added.